mysql-test/suite/innodb/t/percona_bug_1529885.test

```diff
@@ -0,0 +1,14 @@
+--source include/not_windows.inc
```

Comment:
This testcase is a strict subset of innodb.percona_changed_page_bmp_flush, isn't it? If so, we don't need a new testcase